frameworks/react-cra/add-ons/start/assets/src/router.tsx.ejs

```diff
@@ -11,8 +11,7 @@ export const createRouter = () => {
   <% if (addOnEnabled['tanstack-query']) { %>
     const rqContext = TanstackQuery.getContext();
```

Comment:
we must create a new query client for each request. is this happening?

Reply:
I'm honestly not entirely sure how this is wired up

I think this is the implementation

https://github.com/TanStack/create-tsrouter-app/blob/main/frameworks/react-cra/add-ons/tanstack-query/assets/src/integrations/tanstack-query/root-provider.tsx.ejs